lib/signaling/v2/remoteparticipant.js

```diff
@@ -95,7 +96,11 @@ class RemoteParticipantV2 extends RemoteParticipantSignaling {
    */
   addTrack(track) {
     super.addTrack(track);
-    this._getTrackReceiver(track.id).then(track.setTrackTransceiver.bind(track));
+    // NOTE(syerrapragada): DataChannels are rejected in Edge so there
```

Comment:
@syerrapragada Nitpick: "**RTCDataChannels** are not supported in Edge, so there is no **TrackReceiver** for the corresponding **RemoteDataTrackPublication**.